spec/bundle/secure/Gemfile

```diff
@@ -1,4 +1,4 @@
 source 'https://rubygems.org'
 
-gem 'rails', '~> 5.2'
+gem 'rails', '~> 7.0.4'
```

Comment:
need after changing the fixed commit of the advisory db

Reply:
If we need to bump the `rails` version and the `ruby-advisory-db` commit, that can be done in a separate PR which I can immediately merge.